docs/versioned-plugins/integrations/aws-v7.0.0.asciidoc

```diff
@@ -0,0 +1,38 @@
+:plugin: aws
+:type: integration
+:no_codec:
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v7.0.0
+:release_date: 2022-07-25
+:changelog_url: https://github.com/logstash-plugins/logstash-integration-aws/blob/v7.0.0/CHANGELOG.md
+:include_path: ../include/6.x
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="{version}-plugins-{type}s-{plugin}"]
+
+=== AWS Integration Plugin {version}
+
+// include::{include_path}/plugin_header.asciidoc[]
```

Comment:
Before we merge this PR, I'm inclined to make some manual fixes both here and (of course) in the source files.

Another option would be to merge this content as it is, and then fix the inconsistencies in a separate PR. Please let me know if you have a preference. 

Because the plugin and integration headers have been commented out, we're not getting the abbreviated titles (such as v7.0.0) in right nav, and it looks weird. (And always will unless we fix it manually.) 

<img width="379" alt="Screen Shot 2022-07-25 at 5 12 56 PM" src="https://user-images.githubusercontent.com/35154725/180877891-9cbd707a-98e6-4ed5-9897-9daf1ca62c2e.png">

Reply:
Decision: "Merge this content as it is, and then fix the inconsistencies in a separate PR"